src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int height;
+    private final int leg;
+
+    public RightTriangle(String color, int height, int leg) {
+        super(color);
+        this.height = height;
+        this.leg = leg;
+    }
+
+    @Override
+    public void draw() {
+        System.out.print("Figure: RightTriangle, height: " + height + ", leg: " + leg);
+        super.draw();
```

Comment:
The `draw()` method implementation should be conceptually consistent with its purpose. It seems like you are trying to include the area calculation or representation logic here, which should be separate. Consider refactoring this method to only handle the drawing aspect of the figure.